### PR TITLE
Sidepanel

### PR DESCRIPTION
**Pull request by lucadelu**
Removed side panel for Contribute landing page, see #120

### CODE REVIEW

**Comment by veroandreo** [comment]
would you mind including a screenshot for an easier and quick visualization?

File: content/contribute/_index.en.md [hunk 1]
```diff
@@ -1,15 +1,17 @@
 ---
-title: "Get involved"
+title: "Contribute"
```
Comment (veroandreo) [inline content/contribute/_index.en.md:2]: Why did you change this title again? Now the page will have the same title twice, once in the header and again as main title... This was the reason I originally modified it to `Get involved`

Comment (lucadelu) [reply content/contribute/_index.en.md:2]: @veroandreo sorry but I don't understand, now (with last changes) the page look like this 
contribute

It has the same structure of other landing pages

**Comment by lucadelu** [comment]
> would you mind including a screenshot for an easier and quick visualization?

Done, maybe later we can find a more appropriate image if you don't like this

**Comment by neteler** [comment]
I think like this it looks fine and the issue pointed out by @veroandreo is indeed not re-appearing.

**Comment by veroandreo** [comment]
Looks nice! Please, merge :)
